django/conf/urls/__init__.py

```diff
@@ -17,6 +16,8 @@
 
 
 def include(arg, namespace=None, app_name=None):
```

Comment:
If we believe that most users of Django won't be able to reverse-engineer or remember the reason for having two distinct modules — and I do — then we should put everything in the same module.